### ISSUE TITLE
I've got a error.

### ISSUE THREAD

**Issue by marius851000** [HackerExperience/HEBorn#279]
I'll go to test new feature, when It made me a error
![image](https://user-images.githubusercontent.com/22586596/31574118-e2cc1fba-b0c8-11e7-8c14-d2ea0d8ed560.png)
I'm using firefox nightly.

**Comment by kress95** [comment]
It's backend related, so we can't fix it.

**Comment by colindevs** [comment]
Where should we report it, then? It's kinda important, given that it completely blocks people from playing...

**Comment by kress95** [comment]
This is a well-known problem related to helix, we added this error page because the game would be a whitescreen without it.

Most cases of `ERR_PORRA_RENATO` are related to Helix, other cases may be related to browser cache problems, so try cleaning your cache before reporting them (we will eventually block browser cache).

But as we're still stabilizing the API (it's really WIP now), we will get tons of these `ERR_PORRA_RENATO`s.
Try asking on discord about the error before reporting it here...

Ahh, another thing, `DO NOT` report images or texts containing passwords or IPs, _ingame server_ passwords are plain text, so reporting them **will help other people hack you once the game is ready**.

You shouldn't publish them on public channels like github or public discord, if your error contain a password or ip, then either censor the sensitive data or send the error to me or pedro on private channels (like direct messages on disccord), we'll see what we can do about it.

**Note:** I already created an issue regarding sensitive data on errors.
